modules/402-ingress-nginx/crds/ingress-nginx.yaml

```diff
@@ -35,6 +35,7 @@ spec:
 
                     **Caution!** If you set it to "nginx", then Ingress resources lacking the `kubernetes.io/ingress.class` annotation or `spec.ingressClassName` field will also be handled.
                   example: 'nginx'
+                  default: 'nginx'
                   pattern: '^[a-z0-9]([-a-z0-9]*[a-z0-9])?$'
```

Comment:
I guess we also have to change pattern here, according to https://github.com/deckhouse/deckhouse/pull/4932/

Reply:
done, also updated secret name pattern with the same value